src/pages/Hosting/HostingPages/Amenities/Amenities.js

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import AmenitiesOption from './AmenitiesOption';
+import CellWifiIcon from '@mui/icons-material/CellWifi';
+import KitchenIcon from '@mui/icons-material/Kitchen';
+import DirectionsCarIcon from '@mui/icons-material/DirectionsCar';
+import PoolIcon from '@mui/icons-material/Pool';
+import HotTubIcon from '@mui/icons-material/HotTub';
+import OutdoorGrillIcon from '@mui/icons-material/OutdoorGrill';
+
+function Amenities({ newStayInfo }) {
+  const [amenities, setAmenities] = useState([]);
+  const [services, setServices] = useState([]);
+
+  const changeAmenityOptions = id => {
+    if (amenities.includes(id) === false) {
+      setAmenities(prevOptions => [...prevOptions, id]);
+    } else {
+      let newAmenityOption = amenities.filter(number => number !== id);
+      setAmenities(newAmenityOption);
```

Comment:
false해서 걸러네는것 보다 true일때를 생각해서 코드를 짜는 게 다른 사람이 봤을 때도 코드가 잘 읽힐 것 같습니다!
```suggestion
     if (amenities.includes(id)) {
      let newAmenityOption = services.filter(number => number !== id);
      setServices(newAmenityOption);
    } else {
      setServices([...amenities, id]);
    }
```

Reply:
그렇네요! 더불어 ` setAmenities(prevOptions => [...prevOptions, id]);`부분도 저렇게 간략하게 작성해 줄 수 있군요!!